### PR TITLE
LPS-172777 Get optionValue related to listTypeEntry

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://issues.liferay.com/browse/LPS-172777

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java [hunk 1]
```diff
@@ -270,7 +270,8 @@ protected List<String> getValue(String valueString) {
 	protected Portal portal;
 
 	private List<Map<String, String>> _getOptionsFromObjectField(
-		long objectDefinitionId, String objectFieldName) {
+		long objectDefinitionId, String objectFieldName,
+		DDMFormFieldOptions ddmFormFieldOptions) {
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java:274]: @murilostodolni could you sort this alphabetically?

**Comment by murilostodolni** [comment]
ci:stop

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by murilostodolni** [comment]
ci:forward

**Comment by brianchandotcom** [comment]
@carolmariaabb @MirnaGama @murilostodolni merged to master

**Comment by murilostodolni** [comment]
Thank you @brianchandotcom

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**